website/README.md

```diff
@@ -368,7 +404,7 @@ This configuration would display something like the following text on the websit
 A {{ release candidate }} for <Product> {{ v1.0.0 }} is available! The release can be <a href='https://releases.hashicorp.com/<product>/{{ 1.0.0-rc1 }}'>downloaded here</a>.
 ```
 
-You may customize the parameters in any way you'd like. To remove a prerelease from the website, simply delete the `prerelease` parameter from the above component.
+You may customize the parameters in any way you'd like. To remove a prerelease from the website, simply delete the `prerelease` paremeter from the above component.
```

Comment:
Looks like we typo'd here 😧 - no worries on this one but maybe we can fix at the source so it will roll out next time we update?

Reply:
Sounds good - I've made an [Asana task](https://app.asana.com/0/1100423001970639/1200233780102792/f) to track these, and have opened [a PR on nextjs-scripts]() with typo fixes. Will collect typos on that task as [the rest of the README update PRs get reviewed](https://app.asana.com/0/1100423001970639/1200208874632376/f).